tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/Responses/TspToolResponse.cs

```diff
@@ -0,0 +1,38 @@
+using System.Text;
+using System.Text.Json.Serialization;
+
+
+namespace Azure.Sdk.Tools.Cli.Models.Responses
+{
+    public class TspToolResponse : Response
+    {
+        [JsonPropertyName("typespec_project_path")]
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
+        public string TypeSpecProjectPath { get; set; } = string.Empty;
+
+        [JsonPropertyName("is_successful")]
+        public bool IsSuccessful { get; set; }
+
+        [JsonPropertyName("error_message")]
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
+        public string ErrorMessage { get; set; } = string.Empty;
+
+        public override string ToString()
+        {
+            StringBuilder output = new StringBuilder();
+
+            output.AppendLine($"### Is Successful: {IsSuccessful}");
+            if (IsSuccessful)
+            {
+                output.AppendLine($"### TypeSpec Project Path: {TypeSpecProjectPath}");
```

Comment:
Intended to be empty for now?

Reply:
Not sure I follow - `TypeSpecProjectPath` should be filled in if `IsSuccessful` is true - though I'm not sure whether it's better to conditionally include this line or not.